draft-ietf-netmod-schedule-yang.md

```diff
@@ -449,6 +449,22 @@ This section uses the template described in {{Section 3.7 of ?I-D.ietf-netmod-rf
    just for the ease of understanding. Only the message body is provided with
    JSON used for encoding {{?RFC7951}}.
 
+## The "generic-schedule-params" Grouping
+
+   {{ex-0}} indicates the example of a requested schedule that needs to start no earlier than
+   08:00 AM, January 1, 2025 and end no later than 8:00 PM, January 31, 2025 (Beijing time).
+   Schedule requests fail to meet the requirements are ignored by the system.
```

Comment:
```suggestion
   Schedule requests that fail to meet the requirements are ignored by the system.
```

Also, define what's a "system".

Reply:
> Also, define what's a "system".
I think it is a familiar comment, but I couldn't find where we also used the "system" and then defined it properly. My poor memory ;-) 
Would the following text be okay to add into the document where the word "system" is first used:
A system accepts and handles the schedule requests, which may help further automate the scheduling process of events, policy, services, or resources based on date and time.

Reply:
Looks good. Thanks.